db/schema.rb

```diff
@@ -13,7 +13,6 @@
 ActiveRecord::Schema.define(version: 2020_10_24_145705) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_trgm"
```

Comment:
Hey! Oh me should have activated that with a migration probably. Whoopsie

Reply:
Why? Do we still need pg trigrams?